src/index.ts

```diff
@@ -142,9 +146,23 @@ export function withSpid(
     samlConfig
   );
 
-  return loadSpidStrategyOptions()
+  return fromNullable(null)(appConfig.startupIdpsMetadata)
```

Comment:
I'd add some comments to explain what's happening

Reply:
Fixed in a4bdb13